### PR TITLE
Provide better description for the .Net image stream.

### PR DESCRIPTION
**Pull request by jerboaa**
@omajid Please have a look. Thanks!

### CODE REVIEW

File: dotnet_imagestreams.json [hunk 1]
```diff
@@ -19,9 +19,9 @@
                     {
                         "name": "latest",
                         "annotations": {
-                          "description": ".Net Core 1.0 S2I image.",
+                          "description": "Build and run .Net core 1.0 applications",
```
Comment (omajid) [inline dotnet_imagestreams.json]: ".Net core" -> ".NET Core"

Comment (jerboaa) [inline dotnet_imagestreams.json]: Fixed

File: dotnet_imagestreams.json [hunk 2]
```diff
@@ -34,7 +34,7 @@
                     {
                         "name": "1.0",
                         "annotations": {
-                            "description": ".Net Core 1.0 S2I image.",
+                            "description": "Build and run .Net core 1.0 applications",
```
Comment (omajid) [inline dotnet_imagestreams.json]: ".Net core" -> ".NET Core"

Comment (jerboaa) [inline dotnet_imagestreams.json]: Fixed.

**Comment by omajid** [comment]
Looks good to me!

**Comment by jerboaa** [comment]
Thanks!

**Comment by jerboaa** [comment]
[skipci]

**Pull request merged**